embedchain/chunkers/openapi.py

```diff
@@ -0,0 +1,16 @@
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+
+from embedchain.chunkers.base_chunker import BaseChunker
+from embedchain.config.add_config import ChunkerConfig
+
+
+class OpenAPIChunker(BaseChunker):
+    def __init__(self, config: ChunkerConfig | None = None):
```

Comment:
This is why the tests are failing I think.

Reply:
Hmm. But it works locally, thats weird.